packages/react/src/components/form/getFormValues.ts

```diff
@@ -13,14 +13,10 @@
  */
 export const getFormValues = <T extends Result>(form: HTMLFormElement): T =>
   Object.fromEntries(
-    Object.entries(form.elements).filter(hasName).map(elementValue)
-  );
+    (Array.from(form.elements) as HTMLInputElement[])
```

Comment:
If you set this as `HTMLInputElement`, does that work with other form elements?

Reply:
It does, all we need is elements with `.value`, all all `form.elements` should have them.
It's not the most precise or correct interface/type to use, but it's not terribly wrong either.
I suppose we could do `as { value: string }` 🤷🏻

Reply:
Maybe it woul be better?

IDK, just seems that you only expect `<input>` here...

Reply:
Fair enough, created a new interface.